### PR TITLE
Type error in readme

### PR DESCRIPTION
**Pull request by fisx**
(i didn't check if this is compatible with earlier versions of purescript.)

### CODE REVIEW

**Comment by fisx** [comment]
oops, i was too fast.  this change just moves the type error into `spec`.  i'll investigate some more...

**Comment by fisx** [comment]
rebased (sorry).  works beautifully now.

File: test/Main.purs [hunk 1]
```diff
@@ -15,20 +15,20 @@ data Action = Increment | Decrement
 type State = { counter :: Int }
 
 initialState :: State
-initialState = { counter: 0 }
+initialState = { counter: 0.0 }
```
Comment (paf31) [inline test/Main.purs]: The `State` type wraps `Int` here. This won't compile. The README should match what _was_ here.

**Comment by fisx** [comment]
sorry for my clumsiness.  i took the code off the readme and got it to work by substituting everything to `Number`.  should have seen that the type in the example has been changed.

would you like me to squash the last commit?

**Comment by paf31** [comment]
:+1: Thanks!

**Pull request merged**